src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -105,12 +97,12 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
                 {
                     // display disclaimer that this scheme can't be deleted
                     SelectedSchemeDisclaimer().Text(RS_(L"ColorScheme_DeleteDisclaimerInBox"));
-                    SelectedSchemeDisclaimer().Visibility(Visibility::Visible);
+                    SelectedSchemeDisclaimer().Opacity(1);
                 }
                 else
                 {
                     // hide the disclaimer
-                    SelectedSchemeDisclaimer().Visibility(Visibility::Collapsed);
```

Comment:
Note about this: 

`Visibility::Collapsed` means that the element doesn't tell xaml about the space it needs when `Visible`, so what happens when the element changes from `Collapsed` to `Visible` is that the stuff below it shifts down to accommodate the now visible element. I found this to be jarring when using keyboard navigation as the disclaimer might be appearing/disappearing repeatedly, causing the ListView below it to constantly shift down/up.

Using Opacity means that we get the same appearing/disappearing text behaviour, but we always allocate space for the text, to prevent the shifting of the ListView.

Reply:
Good call! Added it in